src/views/product-root-docs-path-landing/components/hero/index.tsx

```diff
@@ -34,7 +34,11 @@ const ProductRootDocsPathLandingHero = ({
 					<Text className={s.pageSubtitle}>{pageSubtitle}</Text>
 				</div>
 			</div>
-			<ProductRootDocsPathLandingIconCardLinkGrid />
+			{currentProduct.slug !== 'terraform' ? (
```

Comment:
Currently the design spec shows no icon cards on the `/terraform/docs` landing view, so that's what I'm aiming for. Chatting with Brayden, it sounds likely that these will get added back in for Terraform in some capacity in the future, but we don't yet have a design spec for which cards to add.

Reply:
thought 💭 : Do we want to make this configurable via the `docs-landing.json` content file? I'm fine with either approach, just curious if you considered that!

Reply:
👆 I think it could be a great option - I hadn't considered that specifically, but did consider maybe using `get-nav-items` like we do in #849 . I ultimately leaned away from it, since it feels like hiding these items is probably a temporary thing, so maybe better to handle it with a one-off conditional rather than introducing a new facet of authorable content. Very open to reconsidering within this PR though, I do like the specificity of putting something in `docs-landing.json`, and I suppose for now we'd do like an empty array to match the design spec? 🤔

Reply:
@zchsh Yeah I was even thinking just a toggle, like `hideCardLinks: true` or something. Also totally fine with leaving it as you have it if it feels like this will be temporary 😄